goldens/skip_lines.in

```diff
@@ -34,3 +34,25 @@ This line should not be sorted.
   Charlie | Baz
   <!-- keep-sorted-test end -->
 <!-- keep-sorted-test end -->
+
+Skip lines with group_prefixes:
+// keep-sorted-test start skip_lines=2 group_prefixes=:
+| birds        |
+| ------------ |
+| blue footed  |
+: booby        |
+| crested      |
+: bobwhite     |
+| anhinga      |
+// keep-sorted-test end
+
+Skip lines with group_prefixes indented:
+// keep-sorted-test start skip_lines=2 group_prefixes=:
+    | birds        |
+    | ------------ |
+    | blue footed  |
+    : booby        |
+    | crested      |
+    : bobwhite     |
+    | anhinga      |
+// keep-sorted-test end
```

Comment:
Oops. Looks like I messed up the merge commit to resolve conflicts. Sorry for the churn :(
```suggestion
// keep-sorted-test end

```

Reply:
That suggestion might not be right...we need the trailing newline back and I'm not entirely sure how to suggest that in GitHub :/

Reply:
Done.